### PR TITLE
Adapt to Foundation website changes

### PR DESCRIPTION
**Pull request by cdietrich**
Adapt to Foundation website changes
- set baseurl to eclipse.dev
- deploy xtend under /Xtext/xtend

See #2706

### CODE REVIEW

**Comment by cdietrich** [comment]
naive test seems to work.
but i assume we also need to adapt a ton of links once dev is live

File: xtext-website/_config.yml [hunk 1]
```diff
@@ -41,11 +41,11 @@ src:
   emf: "https://git.eclipse.org/r/plugins/gitiles/emf/org.eclipse.emf/+/refs/tags/R2_20_0"
 
 javadoc:
-  eclipse-platform: "http://help.eclipse.org/luna/topic/org.eclipse.platform.doc.isv/reference/api"
-  eclipse-jdt: "http://help.eclipse.org/luna/topic/org.eclipse.jdt.doc.isv/reference/api"
-  java: "http://docs.oracle.com/javase/8/docs/api"
+  eclipse-platform: "https://help.eclipse.org/luna/topic/org.eclipse.platform.doc.isv/reference/api"
+  eclipse-jdt: "https://help.eclipse.org/luna/topic/org.eclipse.jdt.doc.isv/reference/api"
```
Comment (LorenzoBettini) [inline xtext-website/_config.yml:45]: As above, https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Findex.html

File: xtext-website/_config.yml [hunk 2]
```diff
@@ -41,11 +41,11 @@ src:
   emf: "https://git.eclipse.org/r/plugins/gitiles/emf/org.eclipse.emf/+/refs/tags/R2_20_0"
 
 javadoc:
-  eclipse-platform: "http://help.eclipse.org/luna/topic/org.eclipse.platform.doc.isv/reference/api"
-  eclipse-jdt: "http://help.eclipse.org/luna/topic/org.eclipse.jdt.doc.isv/reference/api"
-  java: "http://docs.oracle.com/javase/8/docs/api"
+  eclipse-platform: "https://help.eclipse.org/luna/topic/org.eclipse.platform.doc.isv/reference/api"
```
[the remark this answers is not in the document]
Comment (cdietrich) [reply xtext-website/_config.yml:44]: i would do these updates separately, later

File: xtext-website/_config.yml [hunk 3]
```diff
@@ -41,11 +41,11 @@ src:
...
+  eclipse-jdt: "https://help.eclipse.org/luna/topic/org.eclipse.jdt.doc.isv/reference/api"
+  java: "https://docs.oracle.com/javase/8/docs/api"
   javaee: "https://docs.oracle.com/javaee/7/api"
-  junit: "http://junit.sourceforge.net/javadoc"
+  junit: "https://junit.sourceforge.net/javadoc"
```
[the remark this answers is not in the document]
Comment (cdietrich) [reply xtext-website/_config.yml:48]: we also need to check which of thse are still in use

File: Jenkinsfile [hunk 4]
```diff
@@ -71,7 +71,7 @@ spec:
         dir ('git-repo/xtend-website') {
           sh '''
             # generate things in _site
-            bundle exec jekyll build
+            bundle exec jekyll build --destination ../xtext-website/_site/xtend
```
Comment (szarnekow) [inline Jenkinsfile:74]: Nice approach

**Review by szarnekow — APPROVED** [review]

**Review by LorenzoBettini — APPROVED** [review]

**Pull request merged**